     /**
+     * Mine & Blade: Notes
+     * Edits made by nerd-boy: 1
+     * Edit to allow a switch to occur to items that are outside of the normal slot range.
+     */
@@ -565,7 +570,7 @@
      */
     public void handleHeldItemChange(SPacketHeldItemChange packetIn)
     {
-        if (InventoryPlayer.isHotbar(packetIn.func_149385_c()))
+        if (mods.battlegear2.api.core.InventoryPlayerBattle.isValidSwitch(packetIn.func_149385_c()))//-----MB-Edited-----
         {
             this.gameController.thePlayer.inventory.currentItem = packetIn.func_149385_c();
         }
\ No newline at end of file
